src/profile/helpers/__tests__/updateMutationVariables.test.ts

```diff
@@ -134,7 +134,7 @@ test('add arrays are formed correctly', () => {
 
   expect(variables.input.profile.addAddresses).toEqual([
     {
-      address: 'Testikatu 66',
+      address: addresses[1].address,
       postalCode: '00000',
       city: 'Helsinki',
       countryCode: 'FI',
```

Comment:
There's still a lot of repetition even if the tool doesn't complain anymore. Would it be possible to use the `addresses[1]` as a whole (perhaps selecting only wanted fields out of it or something)?

Reply:
The original test seems to be done quick & dirty. There's so much repetition. I've now commited massive changes to the tests. They are not squashed yet, because if it is not satisfactory yet, I don't want to squash multiple times.

Reply:
Dosn't look that massive to me. But cleaner 😃